### PR TITLE
Enable all tests on Windows in CI

### PR DESCRIPTION
**Pull request by Enselic**
I wonder if tests magically work on Windows in CI now given the recent changes to tests?

### CODE REVIEW

**Comment by Enselic** [comment]
Amazingly they did 🤩
Almost seems too good to be true.

@Emilgardis Am I missing something? 🤔 (Asking you since IIRC you tried to debug these failures in the past)

File: cargo-public-api/tests/cargo-public-api-bin-tests.rs [hunk 1]
```diff
@@ -47,7 +47,6 @@ fn create_test_repo_with_dirty_git_tree() -> TestRepo {
 
 // FIXME: This tests is ignored in CI due to some unknown issue with windows
```
Comment (Enselic) [inline cargo-public-api/tests/cargo-public-api-bin-tests.rs]: Oops I should also remove the FIXMEs

**Comment by Emilgardis** [comment]
well well, seems to work now?!

**Comment by Enselic** [comment]
Yep :shrug:

**Pull request merged**